package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "start": "vue-cli-service serve --CATALOG_URL=http://localhost:8080/catalog.json",
     "build": "vue-cli-service build",
-    "build:sit": "vue-cli-service build --CATALOG_URL=https://sit-stac.geoplatform.info/catalog.json --HISTORY_MODE=hash",
-    "build:stg": "vue-cli-service build --CATALOG_URL=https://stg-stac.geoplatform.gov/catalog.json --HISTORY_MODE=hash",
-    "build:prd": "vue-cli-service build --CATALOG_URL=https://stac.geoplatform.gov/catalog.json --HISTORY_MODE=hash",
+    "build:sit": "cross-env STG_ENV='sit' ./build_index.sh && vue-cli-service build --CATALOG_URL=https://sit-stac.geoplatform.info/catalog.json --HISTORY_MODE=hash",
+    "build:stg": "cross-env STG_ENV='stg' ./build_index.sh && vue-cli-service build --CATALOG_URL=https://stg-stac.geoplatform.gov/catalog.json --HISTORY_MODE=hash",
+    "build:prd": "cross-env STG_ENV='prd' ./build_index.sh && vue-cli-service build --CATALOG_URL=https://stac.geoplatform.gov/catalog.json --HISTORY_MODE=hash",
```

Comment:
is cross-env supposed to be a crutch for Windows platforms not running WSL? If so, won't this command still be an issue since you're targeting a shell script?